apps/charterafrica/src/payload/collections/People.js

```diff
@@ -0,0 +1,157 @@
+import { PEOPLE_COLLECTION } from "../../lib/tools/models";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const People = {
+  slug: PEOPLE_COLLECTION,
+  labels: {
+    singular: {
+      en: "People",
+      fr: "People",
+      pt: "Pessoas",
+    },
+    plural: {
+      en: "People",
+      fr: "People",
+      pt: "Pessoas",
+    },
+  },
+  admin: {
+    useAsTitle: "externalId",
+  },
+  access: {
+    read: () => true,
+  },
+  fields: [
+    source(),
+    {
+      name: "externalId",
+      type: "text",
+      required: true,
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "fullName",
+      type: "text",
+      label: {
+        en: "People Full Name",
```

Comment:
```suggestion
        en: "Full Name",
```

Don't repeat collection name.